juniper_official/routing/collect-rib-table-protocol-routes.rule

```diff
@@ -0,0 +1,121 @@
+/*
+ * Collects RIB protocol statistics for plotting and ML
+ */
+ healthbot {
+    topic routing.rib {
+        rule collect-rib-table-protocol-routes {
```

Comment:
no triggers why is this being collected?

Reply:
Added trigger